dist/fabric.js

```diff
@@ -27251,6 +27371,13 @@ fabric.Image.filters.BaseFilter.fromObject = function(object, callback) {
      */
     editingBorderColor: 'rgba(102,153,255,0.25)',
 
+    /**
+     * Border color of text object while indicated
+     * @type String
+     * @default
+     */
+    indicationBorderColor: 'rgba(00,254,0,0.25)',
+
```

Comment:
and this.

Reply:
Yup! Same documentation format as the other vars in the file.